quickstarts/node-js/typescript/config.yml

```diff
@@ -0,0 +1,40 @@
+
+slug: typescript
+description: |
+  ## Comprehensive monitoring quickstart for TypeScript
+  Microsoft developed the TypeScript programming language, a free and open-source project. In addition to adding optional static typing, it is a strict syntactical superset of JavaScript. Its objective is to develop large-scale applications and transform their source code into JavaScript. 
+
+  ## Why monitor TypeScript?
+  - Access your TypeScript logs to identify errors in your application.
+  - Find performance issues before they happen.
+  - Scalable error monitoring without affecting production flow.
+  - Maintain a low latency with fast throughput.
+
+  ## What’s included in this quickstart?
+  Get these TypeScript monitoring features out of the box with our quickstartquickstar:
+  - Understand the health of your with: throughput and error rate data, logs, web transaction time, Apdex score, and more apdex score and more).
+  - Correlate TypeScript application metrics with your server’s primary health metrics, like CPU usage, memory, network usage, and disk space.
+  - Create high-value baseline alerts that proactively inform developers about the status of their apps.
+
+summary: |
+  Instrument and monitor your TypeScript-built app with our Node.js agent. Analyze performance data right out of the box.
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: Typescript
+documentation:
+  - name: Typescript installation docs
+    description: |
+      Instrument and monitor your TypeScript-built app with our Node.js agent. Analyze performance data right out of the box.
+    url: >-
+      https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent/
```

Comment:
Hi @akristen , request you to kindly provide us the right docs url, so that we can update this PR with it. 😃

Reply:
This is the link to view our section on [Typescript](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent#langtech).

Reply:
Thank you.